lib/module.js

```diff
@@ -28,7 +28,7 @@ export default function SentryModule (moduleOptions) {
       ExtraErrorData: {},
       ReportingObserver: {},
       RewriteFrames: {},
-      Vue: { attachProps: true }
+      Vue: { attachProps: true, logErrors: true }
```

Comment:
Could we make it configurable anyway? That should be a quick fix and logging in production could be frowned upon..

Reply:
Technically it is configurable by overriding `clientIntegrations`.
I wouldn't be opposed to adding a configuration option specifically for that if I would have a good place for it. I don't think that adding a new global option for configuration one specific integration would be worth it...

Reply:
I don't really see any problems with posting errors in production as those are client errors and there is nothing to hide there.

But I'm open to setting the default value to `this.options.dev` instead.